actions.go

```diff
@@ -61,6 +61,31 @@ var NotImplementedExecute = func(s *discordgo.Session, m *discordgo.MessageCreat
 }
 
 var (
+	// RootAction is the first action call by Kadok for resolve
+	StatusAction = Action{
+		security.EmptyPermission,
+		"\nJe te dis ce que j'ai dans mon ventre !",
+		map[string]*Action{},
+		func(s *discordgo.Session, m *discordgo.MessageCreate, parameters []string) (string, error) {
+			message := ""
```

Comment:
it may be the right time to introduce go template

Reply:
Issue #36 created